com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalUnlitSubTarget.cs

```diff
@@ -129,23 +127,26 @@ static class SubShaders
                 generatesPreview = true,
                 passes = new PassCollection
                 {
-                    { UnlitPasses.Unlit },
+                    { UnlitPasses.Forward },
                     { CorePasses.ShadowCaster },
                     { CorePasses.DepthOnly },
+                    { UnlitPasses.DebugMaterial },
                 },
             };
 
             public static SubShaderDescriptor UnlitDOTS
             {
                 get
                 {
-                    var unlit = UnlitPasses.Unlit;
+                    var unlit = UnlitPasses.Forward;
```

Comment:
Good question - might have been a mistake, let me check that.

Reply:
Ah, okay - I've renamed it because it's then consistent with the lit shader, we already know it's an unlit pass from the containing class. I can rename it back if you like - it just confused me when I came across it.